src/pages/call/calling.ts

```diff
@@ -0,0 +1,25 @@
+import { Component } from '@angular/core';
+import { NavController, NavParams } from 'ionic-angular';
+import { UserData } from '../../providers/user-data';
+
+@Component({
+  selector: 'page-calling',
+  templateUrl: 'calling.html',
+})
+export class CallingPage {
+    phone_number: string = '';
+
+    constructor(public navCtrl: NavController,
+                public navParams: NavParams,
+                public userData: UserData) {
+       this.phone_number = this.navParams.get('phone_number');
```

Comment:
No, no need to change it. It was more out of curiosity. I was also trying to understand (and googling) when is better to use form vs navparams. Not sure if I should change our topup form now :)